### PR TITLE
kelly/refactor_font_size

### PR DESCRIPTION
**Pull request by kellybinary**
- refactor font-size unit to `rem`
- refactor other unit to `em` or `px`

### CODE REVIEW

File: src/sass/app_2/_common/components/accounts_switcher.scss [hunk 1]
```diff
@@ -43,23 +43,19 @@
             p.acc-balance-amount {
                 line-height: 100%;
                 color: $COLOR_GREEN_1;
-                font-size: 1rem;
-                padding: 0 16px 0 8px;
+                font-size: 1.6rem;
```
Comment (ashkanx) [inline src/sass/app_2/_common/components/accounts_switcher.scss]: same here. why `rem`?

File: src/sass/app_2/_common/components/accounts_switcher.scss [hunk 2]
```diff
@@ -43,23 +43,19 @@
...
-                position: absolute;
-                top: 16px;
-                right: 8px;
-                line-height: 1;
+                line-height: 1.5;
                 transition: transform 0.25s ease;
                 transform: rotate(0deg);
                 transform-origin: 50% 45%;
-                font-size: 1.1rem;
+                font-size: 1.6rem;
```
Comment (ashkanx) [inline src/sass/app_2/_common/components/accounts_switcher.scss]: same here

File: src/sass/app_2/_common/components/accounts_switcher.scss [hunk 3]
```diff
@@ -70,16 +66,15 @@
...
                     content: 'demo';
-                    font-size: 0.75rem;
-                    border: 1px solid $COLOR_ORANGE;
+                    font-size: 1.2rem;
+                    border: 0.1em solid $COLOR_ORANGE;
                     color: $COLOR_ORANGE;
-                    padding: 2px 8px;
-                    border-radius: 16px;
+                    padding: 0.2em 0.8em;
+                    margin-right: 0.8rem;
```
Comment (ashkanx) [inline src/sass/app_2/_common/components/accounts_switcher.scss]: sometimes we use `rem`, sometimes `em` for padding/margin.

File: src/sass/app_2/_common/components/accounts_switcher.scss [hunk 4]
```diff
@@ -113,29 +107,29 @@
...
+        margin: 0.8em 0 1.6em;
 
         span {
-            padding: 8px 16px;
-            border: 1px dashed $COLOR_DARK_BLUE_2;
+            padding: 0.8em 1.6em;
+            border: 0.1em dashed $COLOR_DARK_BLUE_2;
             color: $COLOR_DARK_BLUE_2;
-            font-size: 0.75rem;
+            font-size: 1.2rem;
```
Comment (ashkanx) [inline src/sass/app_2/_common/components/accounts_switcher.scss]: I think in general font-size should be relative to its container, not root.

File: src/sass/app_2/_common/base/common.scss [hunk 5]
```diff
@@ -1,10 +1,36 @@
-body {
-    font-family: $FONT_STACK;
-    position: relative;
-    min-height: 100%;
-    margin: 0;
-    padding: 0;
-    overflow-x: hidden;
+html {
+    font-size: 62.5%; /* 1rem = 10px */
```
[the remark this answers is not in the document]
Comment (kellybinary) [reply src/sass/app_2/_common/base/common.scss]: i think it's easier to convert units in base 10. for any `px` value, we can simply get the `em` or `rem` values by dividing by 10. e.g. 12px will be 1.2em etc.

File: src/sass/app_2/_common/components/accounts_switcher.scss (hunk 4 above)
Comment (kellybinary) [reply src/sass/app_2/_common/components/accounts_switcher.scss]: @ashkanx thanks for the catch, this should be in `em`

File: src/sass/app_2/_common/base/common.scss (hunk 5 above)
[the remark this answers is not in the document]
Comment (ashkanx) [reply src/sass/app_2/_common/base/common.scss]: Understood, Thanks.

[the remark this answers is not in the document]
Comment (negarn) [reply src/sass/app_2/_common/base/common.scss]: @kellybinary can you comment that in the code so we remember it in the future? :D

File: src/sass/app_2/_common/components/accounts_switcher.scss [hunk 6]
```diff
@@ -43,23 +43,19 @@
             p.acc-balance-amount {
                 line-height: 100%;
                 color: $COLOR_GREEN_1;
-                font-size: 1rem;
-                padding: 0 16px 0 8px;
+                font-size: 1.6em;
```
Comment (negarn) [inline src/sass/app_2/_common/components/accounts_switcher.scss]: so we're not going to use variables for fonts?

Comment (kellybinary) [reply src/sass/app_2/_common/components/accounts_switcher.scss]: we will, was thinking to add that in another PR

File: src/sass/app_2/_common/base/common.scss (hunk 5 above)
[the remark this answers is not in the document]
Comment (kellybinary) [reply src/sass/app_2/_common/base/common.scss]: sure! :D

File: src/sass/app_2/_common/layout/header.scss [hunk 7]
```diff
@@ -113,7 +111,7 @@
             justify-content: flex-end;
             order: 1;
             display: inline-flex;
-            border-right: 1px solid $COLOR_LIGHT_BLACK_4;
+            border-right: 0.1em solid $COLOR_LIGHT_BLACK_4;
```
Comment (OskarAhl) [inline src/sass/app_2/_common/layout/header.scss]: should be px?

**Pull request closed without merging**